src/EventStore.Core.Tests/Settings/ReaderThreadCountCalculatorTests.cs

```diff
@@ -5,24 +5,30 @@ namespace EventStore.Core.Tests.Settings {
 	[TestFixture]
 	public class ReaderThreadCountCalculatorTests {
 		[TestCase]
-		public void configured_takes_precedence() => Test(configuredCount: 1, processorCount: 4, expected: 1);
+		public void configured_takes_precedence() => Test(configuredCount: 1, processorCount: 4, false, expected: 1);
```

Comment:
i think it would be nice here to include the parameter name to be consistent with the others (and otherwise the meaning of 'false' isn't immediately obvious)